WeCantSpell.Hunspell/WordList.QuerySuggest.cs

```diff
@@ -2711,28 +2913,26 @@ private string Phonet(string inword)
                 }
             }
 
-            StringBuilderPool.Return(word);
-
             return StringBuilderPool.GetStringAndReturn(target);
         }
 
-        private static void StrMove(StringBuilder dest, int destIndex, StringBuilder src, int srcOffset)
+        private static void StrMove(Span<char> span, int destIndex, int srcOffset)
         {
-            for (var srcIndex = srcOffset; srcIndex < src.Length && destIndex < dest.Length; srcIndex++, destIndex++)
+            for (var srcIndex = srcOffset; srcIndex < span.Length && destIndex < span.Length; srcIndex++, destIndex++)
             {
-                dest[destIndex] = src[srcIndex];
+                span[destIndex] = span[srcIndex];
             }
 
-            if (destIndex < dest.Length)
+            if (destIndex < span.Length)
             {
-                dest[destIndex] = '\0';
+                span[destIndex] = '\0';
             }
         }
 
-#if !NO_INLINE
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-#endif
-        private static void InsertSuggestion(List<string> slst, string word) => slst.Insert(0, word);
+        private static void InsertSuggestion(List<string> slst, string word)
```

Comment:
Is this method worth having? If so, the other usages of `slst.Insert(0, word)` should use this instead.

Reply:
Probably worth keeping, so I used it for other instances of `.Insert(0,` in this context.